### ISSUE TITLE
Broken ``example`` as usually =)

### ISSUE THREAD

**Issue by bashu** [django-fluent/django-fluent-pages#118]
Hey @vdboor 

I am trying to add flatpage object in admin and boom:

``` traceback
ParentAdminNotRegistered at /admin/fluent_pages/page/add/

No parent admin was registered for a '<class 'fluent_pages.models.db.UrlNode'>' model.

Request Method:     POST
Request URL:    http://localhost/admin/fluent_pages/page/add/?ct_id=18&language=en
Django Version:     1.8.14
Exception Type:     ParentAdminNotRegistered
Exception Value:    

No parent admin was registered for a '<class 'fluent_pages.models.db.UrlNode'>' model.

Exception Location:     /Users/bashu/.virtualenvs/fluentpages/lib/python2.7/site-packages/polymorphic/admin/childadmin.py in _get_parent_admin, line 126
Python Executable:  /Users/bashu/.virtualenvs/fluentpages/bin/python
Python Version:     2.7.12
Python Path:    
...
```

**Comment by bashu** [comment]
@vdboor downgraded ``django-polymorpic`` to 0.9.2 and it's working =)

**Comment by vdboor** [comment]
Thanks for reporting! Fixed in a853db44ccd4c40e44a9d686c31b6d547da8a939
